Python/Product/PythonTools/PythonTools/Debugger/DebugAdapter/DebugInfo.cs

```diff
@@ -103,4 +106,23 @@ public class PathRule {
         [JsonProperty("include")]
         public bool? Include { get; set; }
     }
+
+    public class VariablePresentation {
+        [JsonProperty("special")]
+        public string Special { get; set; }
+
+        [JsonProperty("function")]
+        public string Function { get; set; }
+
+        [JsonProperty("class_")]
+        public string Class { get; set; }
+
+        [JsonProperty("protected")]
+        public string Protected { get; set; }
+    }
+
+    public static class PresentationMode {
+        public const string Group = "group";
+        public const string Inline = "inline";
+    }
```

Comment:
If you want to make this stronger typed, enum + `StringEnumConverter` + `JsonConverterAttribute` should do the trick.

Reply:
Done, thanks!